YouTube/scripts/paginate.js

```diff
@@ -0,0 +1,40 @@
+async function pag(event) {
```

Comment:
What is pag? Could you please explain or rename.

Reply:
named it as paginate